### ISSUE TITLE
Run Workflow Missing selected services or mandatory inputs

### ISSUE THREAD

**Issue by slambrechts** [pipecraft2/pipecraft#3]
Hi,

I have selected my working directory which contains one fastq.gz file (after selecting multiplexed / fastq.gz / single end, as file type), added a demultiplex step, and uploaded my index fasta file, which looks like this:

```
>Molenbeek_0_10
AGACAACTC...CGTCCTA
>Herentals_0_10
TTCGGCATT...ATATCGA
>Diepenbeek_0_10
GAATCTAAC...GCGGAGT
>Burdinne_0_10
TTCGAACCT...TCCGAGC
>Cerfontaine_0_10
GAACCAATC...GAACCAATC
```

However, when I try to run the workflow, it says 'Missing selected services or mandatory inputs' (and the button is red)

Quality check however worked fine

Any idea what I am missing here?

**Comment by slambrechts** [comment]
I was able to make it work by using the complete OTU workflow tab, and editing the step there. When building my own workflow by adding single steps, it doesn't seem to work (see above)

**Comment by anslan** [comment]
'**Missing selected services or mandatory inputs**' means that there are some missing mandatory settings, thus the START is disabled. For example, you want to do primer trimming, but have not specified the primer strings; would like to do taxonomy assignment, but have not specified the reference database... Please carefully check if some of the mandatory fields are not filled.

**Comment by slambrechts** [comment]
I know, but for some reason demultiplexing only worked in the complete OTU workflow, and not when added as a single step. Other steps however did work as a single step. So no idea what might have caused this, but I was able to make it work by running the first few steps including demultiplexing inside the complete OTU workflow

**Comment by anslan** [comment]
Should be fixed in v1.0.0
